Wangscape/tilegen/alpha/AlphaCalculatorMode.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace tilegen
+{
+namespace alpha
+{
+
+enum class AlphaCalculatorMode
```

Comment:
Could `Alpha` prefix be removed from this `enum` name? It would become `[tilegen::]alpha::CalculatorMode`, quite clear.